.travis.yml

```diff
@@ -0,0 +1,33 @@
+sudo: false
+language: python
+python:
+  - 3.4
+  - 3.5
+  - 3.6
+
+cache:
+  pip: true
+
+before_install:
+  # Use pip 9, so that setuptools can be downgraded.
+  - pip install pip==9 setuptools==17
+  - pip install -r requirements.txt
+  - pip install -r test-requirements.txt
+
+script:
+  - bash .ci/tests.sh
+  - pip install -e .
+  - pip install coala-bears[alldeps] --pre -U
```

Comment:
My bad, I thought it would be required for running `coala --non-interactive`, didn't look into requirements.txt